File: sams-backend/src/main/java/com/sams/entity/CertificationAuthority.java
package com.sams.entity;

import jakarta.persistence.*;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;
import java.time.LocalDateTime;

@Entity
@Table(name = "m_certification_authority", schema = "base")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class CertificationAuthority {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private Long id;

    @Column(name = "certification_authority_id")
    private Long certificationAuthorityId;

    @Column(name = "org_id")
    private Long orgId;

    @Column(name = "certification_authority_name")
    private String certificationAuthorityName;

    @Column(name = "issuing_authority")
    private String issuingAuthority;

    @Column(name = "created_by")
    private String createdBy;

    @Column(name = "created_dt_disp")
    private String createdDtDisp;

    @Column(name = "updated_by")
    private String updatedBy;

    @Column(name = "updated_dt_disp")
    private String updatedDtDisp;

    @Column(name = "org_name")
    private String orgName;

    @Column(name = "column_name")
    private String columnName;

    @Column(name = "direction")
    private String direction;

    @PrePersist
    protected void onCreate() {
       // Auto-generated
    }

    @PreUpdate
    protected void onUpdate() {
       // Auto-generated
    }
}

File: SAMSUI/src/app/Model/master/certification-authority.ts
import { BaseModel } from '../base/baseModel';

export class CertificationAuthorityModel extends BaseModel{

    public certificationAuthorityId ?: number;

    public orgId ?: number;

    public certificationAuthorityName ?:string;

    public issuingAuthority ?: string;

    public createdBy ?:string;

    public createdDtDisp ?: string;

    public updatedBy ?: string;

    public updatedDtDisp ?: string;

    public pageNumber ?: number;

    public recordsPerPage ?: number;

    public orgName ?: string;

    public columnName ?: string;

    public direction ?: string;

}